tests/test_mlx5_var.py

```diff
@@ -10,6 +10,7 @@
 from tests.base import BaseResources
```

Comment:
Please fix "compare*" typo in the commit message (maybe rewrite as "Instead of string comparison").

Reply:
> Please fix "compare*" typo in the commit message (maybe rewrite as "Instead of string comparison").

Thanks for your review, I'll push a fix soon.